src/app/cube/terms-of-service/terms-of-service.component.html

```diff
@@ -91,6 +91,46 @@ <h3 tabindex="0">Changes in Service</h3>
       without providing prior notice. We also retain the right to create limits on your use of the Service
       including storage, at our sole discretion, at any time without prior notice to you.</p>
 
+    <h3>Cookies</h3>
+    <p>
+      We use cookies and other technologies on our application for functionality
+      and performance. The first use of cookies on CLARK is for authorization
+      purposes, this allows us to refer to your account information quick and
+      easily. This is also what makes it possible for you to browse our
+      application without having to login between each refresh or new page. The
+      cookie stored is created by us and no third party is used, the information
```

Comment:
I would remove the "and no third part is used"

Reply:
We need to make this explicit so it should be left.